src/modules/reporting/actions/submit-initial-report.js

```diff
@@ -28,3 +29,29 @@ export const submitInitialReport = (marketId, selectedOutcome, invalid, history,
     })
   }
 }
+
+
+export const estimateSubmitInitialReport = (marketId, callback = logError) => (dispatch, getState) => {
```

Comment:
Please add test coverage

Reply:
It simple server call no logic to test

Reply:
ok, will write unit test for logic based on service call success or fails. makes sense

Reply:
done unit tests written